packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -76,27 +48,13 @@ export const upgradeVaults = async (powers, { options }) => {
    *   Installation<import('../../src/vaultFactory/vaultFactory.js')['start']>
    * >}
    */
-  let installationP;
+  const installationP = E(zoe).installBundleID(bundleID);
+  produceVaultInstallation.reset();
+  produceVaultInstallation.resolve(installationP);
```

Comment:
The `publishRef` / `install` stuff in `builders/scripts/*.js` is supposed to take care of this `installBundleId` and `reset` / `resolve`.

But the only cost to doing it this way is 1 cross-vat call and a bit of allocation / GC. And there's benefit in clarity.

And it looks like the `reset` is missing from the other code path:

https://github.com/Agoric/agoric-sdk/blob/aedfac6ea5fd276a4f64a327cad18d3633901547/packages/deploy-script-support/src/coreProposalBehavior.js#L169-L171